tests/local_settings.example.py

```diff
@@ -1,7 +1,7 @@
 # RENAME THIS FILE TO local_settings.py IF YOU NEED TO CUSTOMIZE SOME SETTINGS
 # BUT DO NOT COMMIT
 
-#DATABASES = {
+# DATABASES = {
```

Comment:
cancel changes to this file, and include this file to the excluded files of flake8 and isort